packages/core/lib/services/substrate/service.js

```diff
@@ -47,7 +47,10 @@ export class SubstrateService {
       amount,
     );
     const paymentInfo = await extrinsic.paymentInfo(account);
-    return paymentInfo.partialFee.toNumber() / DOCK_TOKEN_UNIT;
+    return (
+      paymentInfo.partialFee.muln(110).div(BN_HUNDRED).toNumber() /
```

Comment:
what's the 110 and BN_HUNDRED?

Can we get a test added that protects against this issue in the future?

Reply:
it's adding 10% to the actual fee, this is the logic used by polkadot fe

Reply:
Yes let me add a test to it

Reply:
Can we add a constant like, FEE_ESTIMATION_BUFFER to make it clear what it is? Also could you achieve the same thing by doing a simple `muln(1.10)` and save an operation? It would be clearer to me as a decimal what you're doing